json_schema/decimal.yaml

```diff
@@ -1,5 +1,5 @@
 $schema: https://json-schema.org/draft/2020-12/schema
-$id: https://example.com/product.schema.json
+$id: https://aep.dev/json_schema/decimal
```

Comment:
It sounds like although that's not strictly required, it might be nice to match up with the retrieval URL: https://json-schema.org/understanding-json-schema/structuring#retrieval-uri

Reply:
no. JSON Schema `$id` is only an identifier for the implementation to load the schema into a registry during validation, it doesn't have any expectation of retrieval from this uri. If you wanted to serve this schema from the same uri, it SHALL be served with content-type `application/schema+json`.